src/Teapot.Web.Tests/UnitTests/SuppressBodyTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Teapot.Web.Controllers;
+using Teapot.Web.Models;
+using Teapot.Web.Models.Unofficial;
+
+namespace Teapot.Web.Tests.UnitTests;
+public class SuppressBodyTests {
+    private TeapotStatusCodeResults _statusCodes;
+
+    [SetUp]
+    public void Setup() {
+        _statusCodes = new(
+            new AmazonStatusCodeResults(),
+            new CloudflareStatusCodeResults(),
+            new EsriStatusCodeResults(),
+            new LaravelStatusCodeResults(),
+            new MicrosoftStatusCodeResults(),
+            new NginxStatusCodeResults(),
+            new TwitterStatusCodeResults()
+        );
+    }
+
+    [Test]
+    [TestCase(true)]
+    [TestCase(false)]
+    [TestCase(null)]
+    public void SuppressBodyReadFromQuery(bool? suppressBody) {
+        StatusController controller = new(_statusCodes) {
+            ControllerContext = new ControllerContext {
+                HttpContext = new DefaultHttpContext()
+            }
+        };
+
+        IActionResult result = controller.StatusCode(200, null, suppressBody);
+
+        Assert.Multiple(() => {
+            Assert.That(result, Is.InstanceOf<CustomHttpStatusCodeResult>());
+
+            CustomHttpStatusCodeResult r = (CustomHttpStatusCodeResult)result;
+            Assert.That(r.SuppressBody, Is.EqualTo(suppressBody));
+        });
+    }
+
+    [Test]
+    [TestCase("true")]
+    [TestCase("false")]
+    [TestCase("")]
+    public void SuppressBodyReadFromHeader(string? suppressBody)
+    {
+        StatusController controller = new(_statusCodes)
+        {
+            ControllerContext = new ControllerContext
+            {
+                HttpContext = new DefaultHttpContext()
+            }
+        };
```

Comment:
I'm not bothered which for of indenting is used here vs `SuppressBodyReadFromQuery`, but can we have them consistent?